haiku-libs/haikuwebkit/haikuwebkit-1.2.0.recipe

```diff
@@ -60,38 +69,21 @@ BUILD_PREREQUIRES="
 	cmd:ruby
 	"
 
-SOURCE_DIR="haikuwebkit_x86-$portVersion-Source"
-
 BUILD()
 {
 	#cmake -DPORT=Haiku  \
 	#	-DCMAKE_INSTALL_PREFIX=$prefix -DLIB_SUFFIX=/`echo $secondaryArchSuffix | tr -d _`
 	#make
 	Tools/Scripts/build-webkit --haiku --no-webkit2 \
-		--cmakearg="-DCMAKE_LIBRARY_PATH=`finddir B_SYSTEM_DEVELOP_DIRECTORY`/lib -DCMAKE_INSTALL_PREFIX=$prefix"
+	--cmakearg="-DCMAKE_LIBRARY_PATH=`finddir B_SYSTEM_DEVELOP_DIRECTORY`/lib -DCMAKE_INSTALL_PREFIX=$prefix"
 }
 
 INSTALL()
 {
 	cd WebKitBuild/Release
 	make install
-
 	prepareInstalledDevelLibs libWebKit
-
 	# devel package
 	packageEntries devel \
-		$developDir
+	$developDir
```

Comment:
This and the BUILD section were correct the way they were before; don't modify them.